transaction/core/src/fog_hint.rs

```diff
@@ -109,7 +109,7 @@ impl FogHint {
         ingest_server_private_key: &RistrettoPrivate,
         ciphertext: &EncryptedFogHint,
     ) -> Result<Self, CryptoBoxError> {
-        let plaintext = VersionedCryptoBox::default()
+        let (_result, plaintext) = VersionedCryptoBox::default()
```

Comment:
Can you please explain why it is okay to ignore a failed decryption and relying on the MAGIC_NUMBER check?

Reply:
Good catch.  The magic number check was also failing, but better to return the right error.

Fixed.